# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/mochitest/tree/test_list.html
# Commit: 5ff1cc0ecdb3
# Full Hash: 5ff1cc0ecdb3cc503d6e0b6f5509eda78d551c7a
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2021-05-05 04:13:36
# Regressor Bug: 1513447
# File Overlap Count: 1
# Description:
#   Bug 1513447 - Use generated marker elements for list bullet accessibles. r=Jamie
#   
#   In 1513447 there is a demonstrated instance in which the generated
#   marker is replaced with another one and throws the list item bullet
#   state into an unknown state. To remedy this we need to observe when such
# ==============================================================================

diff -r 029a5e8fa8f3 -r 5ff1cc0ecdb3 accessible/tests/mochitest/tree/test_list.html
--- a/accessible/tests/mochitest/tree/test_list.html	Tue May 04 18:46:44 2021 +0000
+++ b/accessible/tests/mochitest/tree/test_list.html	Tue May 04 19:07:59 2021 +0000
@@ -188,13 +188,9 @@
           { // li
             role: ROLE_LISTITEM,
             children: [
-              { // ::marker content text
-                role: ROLE_STATICTEXT,
-                name: "foo",
-              },
-              { // ::marker content counter
-                role: ROLE_STATICTEXT,
-                name: "1",
+              { // ::marker content text and counter
+                role: ROLE_LISTITEM_MARKER,
+                name: "foo 1",
               },
               {
                 role: ROLE_TEXT_LEAF,
@@ -205,13 +201,9 @@
           { // li
             role: ROLE_LISTITEM,
             children: [
-              { // ::marker content text
-                role: ROLE_STATICTEXT,
-                name: "foo",
-              },
-              { // ::marker content counter
-                role: ROLE_STATICTEXT,
-                name: "2",
+              { // ::marker content text and counter
+                role: ROLE_LISTITEM_MARKER,
+                name: "foo 2",
               },
               {
                 role: ROLE_TEXT_LEAF,